automated-comments/csharp/general/use_private_visibility.md

```diff
@@ -0,0 +1 @@
+As the `%{name}` field is only used within its class, its visibility can be set to `private`.
```

Comment:
@mikedamay said:

“field is only used within its class, its visibility can be set to private.” Maybe “its visibility can and, almost always should, be set to private.”?

Reply:
Good point! I've updated.